src/dotty/tools/backend/jvm/CollectSuperCalls.scala

```diff
@@ -17,14 +17,14 @@ class CollectSuperCalls extends MiniPhaseTransform {
 
   def phaseName: String = "collectSuperCalls"
 
-  override def transformSuper(tree: Super)(implicit ctx: Context, info: TransformerInfo): Tree = {
+  override def transformSelect(tree: tpd.Select)(implicit ctx: Context, info: TransformerInfo): tpd.Tree = {
```

Comment:
Usually we do `import tpd._` at the beginning of a class to avoid adding the prefix everywhere: https://github.com/lampepfl/dotty/blob/master/src/dotty/tools/dotc/transform/ClassOf.scala#L20